packages/material-ui/src/index.js

```diff
@@ -56,7 +59,7 @@ export { default as ExpansionPanelSummary } from './ExpansionPanelSummary';
 export { default as Fab } from './Fab';
 export { default as Fade } from './Fade';
 export { default as FilledInput } from './FilledInput';
-export { default as FormControl } from './FormControl';
+export { default as FormControl, useFormControl } from './FormControl';
```

Comment:
Should `useFormControl` be public?

Reply:
It's already public on the second level

Reply:
I believe that the current official rule is to consider a module public if he's exposed to the barrel index.js. But it makes sens. If it was exported in the second level, we likely intended to make it exported in the main level.